.github/workflows/e2e-test-ci-v2-dev.yml

```diff
@@ -0,0 +1,215 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: e2e-test-ci (dev)
+
+on:
+  push:
+    branches:
+      - master
+      - v1.6.0
+  pull_request:
+    branches:
+      - master
+      - v1.6.0
+    types: [opened, synchronize, reopened, review_requested]
```

Comment:
I see that CI has passed.  So now you can modify this trigger condition to schedule by time (we can run it everyday)